AdminWebsite/AdminWebsite.UnitTests/Mappers/HearingUpdateRequestMapperTest.cs

```diff
@@ -27,17 +34,52 @@ public void Should_map_properties_for_update_hearing_request()
                 QuestionnaireNotRequired = false,
                 AudioRecordingRequired = false
             };
+        }
 
-            var result = HearingUpdateRequestMapper.MapTo(source, username);
+        [Test]
+        public void Should_map_properties_for_update_hearing_request()
+        {
+            var result = HearingUpdateRequestMapper.MapTo(_newParticipantRequest, _username);
 
-            result.HearingRoomName.Should().Be(source.HearingRoomName);
-            result.HearingVenueName.Should().Be(source.HearingVenueName);
-            result.ScheduledDateTime.Should().Be(scheduledDateTime);
-            result.ScheduledDuration.Should().Be(source.ScheduledDuration);
-            result.OtherInformation.Should().Be(source.OtherInformation);
-            result.Cases.Should().BeEquivalentTo(caseRequest);
+            result.HearingRoomName.Should().Be(_newParticipantRequest.HearingRoomName);
+            result.HearingVenueName.Should().Be(_newParticipantRequest.HearingVenueName);
+            result.ScheduledDateTime.Should().Be(_scheduledDateTime);
+            result.ScheduledDuration.Should().Be(_newParticipantRequest.ScheduledDuration);
+            result.OtherInformation.Should().Be(_newParticipantRequest.OtherInformation);
+            result.Cases.Should().BeEquivalentTo(_caseRequest);
             result.QuestionnaireNotRequired.Should().BeFalse();
             result.AudioRecordingRequired.Should().BeFalse();
         }
+
+        [Test]
+        public void Should_Be_true_for_when_linkedParticipant_is_interpreter()
+        {
+            _newParticipantRequest.Participants = new List<EditParticipantRequest>
+            {
+                new EditParticipantRequest
+                {
+                    LinkedParticipants = new List<LinkedParticipant>
+                    {
+                        new LinkedParticipant
+                        {
+                            Id = Guid.NewGuid(),
+                            LinkedId = Guid.NewGuid(),
+                            ParticipantId = Guid.NewGuid(),
+                            Type = LinkedParticipantType.Interpreter
+                        }
+                    }
+                }
+            };
+            var result = HearingUpdateRequestMapper.MapTo(_newParticipantRequest, _username);
+
+            result.HearingRoomName.Should().Be(_newParticipantRequest.HearingRoomName);
+            result.HearingVenueName.Should().Be(_newParticipantRequest.HearingVenueName);
+            result.ScheduledDateTime.Should().Be(_scheduledDateTime);
+            result.ScheduledDuration.Should().Be(_newParticipantRequest.ScheduledDuration);
+            result.OtherInformation.Should().Be(_newParticipantRequest.OtherInformation);
+            result.Cases.Should().BeEquivalentTo(_caseRequest);
+            result.QuestionnaireNotRequired.Should().BeFalse();
```

Comment:
I think this should be 

result.QuestionnaireNotRequired.Should().Be(_newParticipantRequest.QuestionnaireNotRequired);